content/docs/get-started/azure/deploy-changes.md

```diff
@@ -12,53 +12,101 @@ menu:
 aliases: ["/docs/quickstart/azure/deploy-changes/"]
 ---
 
-Now let's deploy our changes.
+Deploy your changes by using `pulumi up` again.
 
 ```bash
 $ pulumi up
 ```
 
-Pulumi computes the minimally disruptive change to achieve the desired state described by the program.
+Pulumi will run the `preview` step of the update, which computes the minimally disruptive change to achieve the desired state described by the program.
 
 ```
 Previewing update (dev):
 
-     Type                      Name                     Plan       Info
-     pulumi:pulumi:Stack       quickstart-dev
- ~   └─ azure:storage:Account  storage                  update     [diff: ~tags]
+     Type                                                         Name                     Plan
+     pulumi:pulumi:Stack                                          quickstart-dev
+ +   ├─ azure-native:storage/latest:StorageAccountStaticWebsite   staticWebsite            create
+ +   └─ azure-native:storage/latest:Blob                          index.html               create
```

Comment:
```suggestion
 +   └─ azure-native:storage:Blob                          index.html               create
```

`latest` will not show in resource type names in Azure-Native (applies everywhere)